metrics-core/src/main/java/com/codahale/metrics/ScheduledReporter.java

```diff
@@ -291,8 +291,17 @@ protected boolean isShutdownExecutorOnStop() {
         return shutdownExecutorOnStop;
     }
 
-    protected Set<MetricAttribute> getDisabledMetricAttributes() {
-        return disabledMetricAttributes;
+    /**
+     * Check if Attribute disabled to be logged
+     * @param attribute metric attribute
+     * @return True if attribute is configured to be disabled
+     */
+    protected boolean isMetricAttributeDisabled(MetricAttribute attribute) {
```

Comment:
I don't believe we need to extract the logic of checking of an attribute to a separate method. `disabledMetricAttributes` can't be null, so we basically have one method which does nothing but delegates a call to the `contains` method of `EnumSet`.